scheduler/node/node.go

```diff
@@ -0,0 +1,86 @@
+package node
```

Comment:
the only way I see (but I don't really like) is to move the strategies and filters into the scheduler package itself. but it fee less clean.

`Node` even though it's a package, it's nested in `scheduler/` so it's not so bad.